app/lin_alg_problems.py

```diff
@@ -145,7 +145,43 @@
         {"test": "transpose_matrix([[1,2],[3,4],[5,6]])", "expected_output": "[[1, 3, 5], [2, 4, 6]]"},
         {"test": "transpose_matrix([[1,2,3],[4,5,6]])", "expected_output": "[[1, 4], [2, 5], [3, 6]]"}
     ]
-}, "Calculate Mean by Row or Column (easy)":{
+}, "Reshape Matrix (easy)": {
+    "description": "Write a Python function that reshapes a given matrix into a specified shape.",
+    "example": """Example:
+    input: a = [[1,2,3,4],[5,6,7,8]], new_shape = (4, 2)
+    output: [[1, 2], [3, 4], [5, 6], [7, 8]]
+    reasoning: The given matrix is reshaped from 2x4 to 4x2.""",
+    "video": "https://youtu.be/DNoLs5tTGAw?si=vpkPobZMA8YY10WY",
```

Comment:
Change the link from the url to "coming soon"

Reply:
Updated the URL.